client/src/components/Home.tsx

```diff
@@ -223,75 +223,69 @@ function Home() {
                 }
                 @media (prefers-color-scheme: dark) {
                   :hover {
-                    filter: brightness(110%);
+                    filter: brightness(140%);
                   }
                 }
               `}
             >
-              <div
+              <Box
+                key={p.id}
                 className={css`
-
-                  display: flex;
-                  flex-direction: column;
-                  align-items: center;
-                  margin: 0 0 2rem 0;
-                  width: 100%;
-                  overflow: hidden;
-                  white-space: ellipsis;
-                  outline: solid 1px grey;
-                  transition: 0.2s ease-in-out;
-                  background-color: var(--mi-light-background-color);
-
-                  > :last-child {
-                    color: var(--mi-primary-color) !important;
-                    background: var(--mi-secondary-color);
-                  }
-
-                  :hover {
-                    background-color: var(--mi-normal-background-color);
-                    transition: 0.2s ease-in-out;
-                    > :last-child {
-                      background: var(--mi-primary-color);
-                      color: var(--mi-secondary-color) !important;
-                    }
-                  }
-
+                  border: solid 1px grey;
+                  margin-bottom: 0.5rem !important;
+                  padding: 0 !important;
+                  justify-content: space-between;
                   @media screen and (max-width: ${bp.medium}px) {
-                    padding: 0rem !important;
+                    width: 100% !important;
+                  }
+                  @media screen and (max-width: ${bp.small}px) {
+                    font-size: var(--mi-font-size-small) !important;
                   }
                 `}
               >
-                <Box
-                  key={p.id}
+                <div
                   className={css`
-                    @media screen and (max-width: ${bp.medium}px) {
-                      width: 100% !important;
-                      padding: 2rem 2rem !important;
+                    display: flex;
```

Comment:
Feel like we need to institute a rule that if a CSS class is more than X lines it should be turned into a component. Wanna say 5 personally. Seriously impacts readability of the component, as well as confusing functionality with style.